docs/how-to/build/web.md

```diff
@@ -6,5 +6,141 @@ grand_parent: How to
 ---
 
 # Build ONNX Runtime for Web
+{: .no_toc }
 
- *[COMING SOON]* 
+There are 2 steps to build ONNX Runtime Web:
+
+- build ONNX Runtime for WebAssembly
```

Comment:
- build ONNX Runtime for WebAssembly
   - or skip and download a pre-built artifacts

People may find out that they can download after they try to build.